pycqed/measurement/calibration/calibration_points.py

```diff
@@ -40,7 +47,9 @@ def create_segments(self, operation_dict, pulse_modifs=dict(),
                     if k == 0:
                         pulse['ref_pulse'] = 'segment_start'
                     if len(pulse_modifs) > 0:
-                        pulse = sweep_pulse_params([pulse], pulse_modifs)[0][0]
+                        pulse = sweep_pulse_params(
+                            [pulse], pulse_modifs,
+                            pulse_not_found_warning=False)[0][0]
```

Comment:
why is the warning disabled here? Is it expected that the pulse is often not found?